platform/solr/solr-schema/src/main/resources/solr/conf/solrconfig.xml

```diff
@@ -1290,6 +1290,7 @@
       <str name="sortField">ext.population_lng</str>
       <str name="payloadField">id_txt</str>
       <str name="buildOnStartup">false</str>
+      <str name="highlight">false</str>
```

Comment:
❗ I think this suggest component can only be used for gazetteer suggestions as it is.  Since this PR is changing how that works, I think it would be safe to completely re-purpose this suggest component for the gazetteer collection. The request handler below could be mapped to `/gazetteer`.  The `serachComponent` name could be changed from `suggest` to `gazetteer`. The `field`, `contextField`, `weightField`, and `sortField` could all be updated to use gazetteer specific field names.

Reply:
@pklinef wouldn't changing that break the other usages where things are using the suggester (like [here](https://github.com/codice/ddf/blob/master/catalog/solr/catalog-solr-core/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java#L273))

Reply:
I think that method is only used by gazetteer plugins.

https://github.com/codice/ddf/search?q=SUGGESTION_QUERY_KEY&unscoped_q=SUGGESTION_QUERY_KEY

We could keep the old one and the new one (since I suggest changing the context path) if we wanted.  But I do not know what the `field`, `contextField`, `weightField`, and `sortField` should be for a generic catalog metacard suggester.  That is why I suggested just replacing it.

Reply:
Added independent request handler